pkgs/development/python-modules/laspy/default.nix

```diff
@@ -0,0 +1,36 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, numpy
+, laszip
+, pytestCheckHook
+}:
+
+buildPythonPackage rec {
+  pname = "laspy";
+  version = "2.3.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-Wdbp6kjuZkJh+pp9OVczdsRNgn41/Tdt7nGFvewcQ1w=";
+  };
+
+  propagatedBuildInputs = [
+    numpy
+    laszip
+  ];
+
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ pname "laszip" ];
```

Comment:
No, because laszip is a runtime dependency that changes the conditional import behavior of laspy if it's available. It's best to include it in order to test the program is going to work correctly.